src/strings.ts

```diff
@@ -4,14 +4,24 @@ export class Strings {
 		'Error.4XX': 'You no longer have access to this notebook.',
 		'Error.5XX': 'Something went wrong on our end.',
 		'Error.Fallback': 'Something went wrong.',
+		'Error.ValidateNotebookName.NotebookNameDotMessage': 'Notebook name cannot begin or end with a dot.',
```

Comment:
How does this get localized?

Reply:
These represent default string fallbacks. The locale overrides are passed as overrides into the component as props.